app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError, VaccineError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    not_wearing_mask = []
+    counter = 0
```

Comment:
And why do you need the list above? We can count masks that are needed to be bought using just 1 variable (and name it using descriptive name, not just `counter`